src/workingWithData.ts

```diff
@@ -0,0 +1,54 @@
+import { v4 as uuidv4 } from 'uuid'
+import { resolve } from 'path'
+import { readFile, open, writeFile, unlink, access, mkdir } from 'fs/promises'
+
+type UserInfoType = { name: string; age: number }
+
+export const createFolderIfNotExist = async function (path: string) {
+  const pathToDIr: string = resolve(path)
+  try {
+    await access(pathToDIr)
+  } catch (error) {
+    await mkdir(pathToDIr)
+    console.log('The dir has been created')
+  }
+}
+
+export const createContent = async function (
+  path: string,
+  fileName: string,
+  content: UserInfoType
+) {
+  const takePath: string = resolve(path, fileName)
+
+  try {
+    await access(takePath)
+    await writeFile(takePath, JSON.stringify(content))
+    console.log('Content has been added')
+  } catch (error) {
+    await open(takePath, 'a+')
+    console.log('File was created')
+  }
+}
+
+export const getContent = async function (path: string, fileName: string) {
+  const filePath: string = resolve(path, `${fileName}`)
+
+  try {
+    const fileContent: string = await readFile(filePath, 'utf-8')
```

Comment:
You don't use this variable in you code below. Please return value, if reading file was succesfull
https://nodejs.org/api/fs.html#fs_filehandle_readfile_options

Reply:
Not done))))) 
![image](https://user-images.githubusercontent.com/67024726/115985178-5644a380-a5b3-11eb-96c0-a858fd706460.png)
![image](https://user-images.githubusercontent.com/67024726/115985193-5fce0b80-a5b3-11eb-9d45-80e928b77fcc.png)
Vs code bagged and creating dfaf dir every time when i start. It is using my last code which doesn't exists now........

Reply:
Remove vs Code and use normal IDE or find place where you create dir dfaf